modules/images/src/lib/parsed-image-api/image-type.js

```diff
@@ -1,42 +1,48 @@
 /* global ImageBitmap, Image */
 import {global, isBrowser} from '../utils/globals';
-import assert from '../utils/assert';
 
-export const IMAGE_BITMAP_SUPPORTED = typeof ImageBitmap !== 'undefined';
-export const HTML_IMAGE_SUPPORTED = typeof Image !== 'undefined'; // NOTE: "false" positives if jsdom is installed
-export const NODE_IMAGE_SUPPORTED = Boolean(global._parseImageNode);
+const IMAGE_SUPPORTED = typeof Image !== 'undefined'; // NOTE: "false" positives if jsdom is installed
+const IMAGE_BITMAP_SUPPORTED = typeof ImageBitmap !== 'undefined';
+const NODE_IMAGE_SUPPORTED = Boolean(global._parseImageNode);
 
 // Checks if a loaders.gl image type is supported
 export function isImageTypeSupported(type) {
   switch (type) {
     case 'auto':
-      return true;
+      // Should only ever be false in Node.js, if polyfills have not been installed...
```

Comment:
maybe notify the users that they could potentially install `@loaders.gl/polyfill` package?